app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player

```

Comment:
![Screenshot from 2023-10-21 17-41-12](https://github.com/mate-academy/py-elves-and-dwarves/assets/122517203/7c4ce871-f62a-47f0-9789-247e08a8e7a5)